staging/src/k8s.io/apiserver/plugin/pkg/audit/safe/safe.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package safe
```

Comment:
What do you mean by embedding this in options? Wouldn't it require handling this flag in every backend?

Reply:
I meant just append https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/options/audit.go with:

```
type safeBackend struct {
	audit.Backend
}

func (s *safeBackend) ProcessEvents(ev ...*auditinternal.Event) bool {
	s.Backend.ProcessEvents(ev...)
	return true
￼}
```

Since it's only used in that file anyway.

Reply:
Geez, you're right, that makes way more sense. Thanks, updated.